models/Document/PageSnippet.php

```diff
@@ -607,4 +619,46 @@ public function getUrl($hostname = null, $scheme = null)
 
         return $url;
     }
+
+    /**
+     * @return int|null
+     */
+    public function isContentRequired()
+    {
+        return $this->contentRequired;
```

Comment:
The property `contentRequired` isn't persisted to the database, as the database column is called `requireContent` so the auto-mapping doesn't work. So we need to unify the naming everywhere, I'd anyway prefer a name like `hasRequiredEditables` or something similar, which is more self-explaining. 

Please change that for all methods, properties, column names, variables, ... Thanks!

Reply:
changed to requireEditableValues and is consistent now with db.